### PR TITLE
[12.0][FIX] currency_rate_update: Fix security vulnerability from CWE-61

### PR DESCRIPTION
**Pull request by abhinavvaidya-tsd**

### CODE REVIEW

File: currency_rate_update/models/res_currency_rate_provider_ECB.py [hunk 1]
```diff
@@ -89,3 +91,8 @@ def startElement(self, name, attrs):
                     (self.date_to is None or self.date <= self.date_to) and \
                     currency in self.currencies:
                 self.content[self.date.isoformat()][currency] = rate
+
+def parse_ecb_response(response, handler):
```
Comment (pedrobaeza) [inline currency_rate_update/models/res_currency_rate_provider_ECB.py]: Please don't make general methods outside the class, and anyway, being only 3 lines and not being reusable due to the scope, just put these 3 lines in the previous method.

**Review by pedrobaeza — COMMENTED** [review]
Thanks for the patch.

**Comment by abhinavvaidya-tsd** [comment]
hi @pedrobaeza I have created the PR to fix issue#177 but I do not know why it fails:

"ERROR: Could not find a version that satisfies the requirement pylint-plugin-utils==0.7 (from -r /home/travis/maintainer-quality-tools/travis/pylint_odoo_requirements.txt (line 1)) (from versions: 0.1, 0.1.1, 0.2, 0.2.1, 0.2.2, 0.2.3, 0.2.4, 0.2.6, 0.3, 0.4, 0.5, 0.6)
ERROR: No matching distribution found for pylint-plugin-utils==0.7 (from -r /home/travis/maintainer-quality-tools/travis/pylint_odoo_requirements.txt (line 1))
The command "travis_install_nightly" failed and exited with 1 during ."

**Comment by pedrobaeza** [comment]
12.0 version is too old and CI is not adapted to "new times". One option is to use copier for updating the template for getting new CI definitions. Other is to make the PR to newer versions.

**Comment by abhinavvaidya-tsd** [comment]
> 12.0 version is too old and CI is not adapted to "new times". One option is to use copier for updating the template for getting new CI definitions. Other is to make the PR to newer versions.

I tried adding in the .travis.yml but I see that I not the correct way to do it as it still fails :( I tried to look for copier and installed it too but did not know how to make it work.

**Comment by pedrobaeza** [comment]
I have fixed the CI in #179. Please rebase and use proper commit message:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#71commit-message

**Comment by abhinavvaidya-tsd** [comment]
@pedrobaeza thank you, I have fixed the PR with appropriate message and it is passing all the checks now.

**Comment by pedrobaeza** [comment]
Module is not `currency`, but `currency_rate_update`, but let's not block this more time. Another thing is that it's not needed to bump manifest version. It's done on merge. I'm going to merge directly for changing commit message.

**Comment by pedrobaeza** [comment]
Please forward-port the change to upper versions.

**Pull request merged**